SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+type UnrealRPCPayload {
+    uint32 offset = 1;
+    uint32 rpc_index = 2;
+    bytes rpc_payload = 3;
+}
+
+type Void {}
```

Comment:
Nit: can you make the braces match the other empty types we have: the closing brace is on the second line.

Reply:
Done